.ondevice/index.tsx

```diff
@@ -1,13 +1,26 @@
+import React from 'react';
 import AsyncStorage from '@react-native-async-storage/async-storage';
 import { view } from './storybook.requires';
+import { SafeAreaView } from 'react-native';
+
+let onDeviceUI = true;
```

Comment:
it could probably just be 
`const onDeviceUI = !process.env.EXPO_PUBLIC_STORYBOOK_SNAPSHOT`